config/fxdata/objects.cfg

```diff
@@ -167,6 +167,17 @@ Persistence = 0
 Immobile = 1
 Properties = EXISTS_ONLY_IN_ROOM BEATING HEART
 UpdateFunction = UPDATE_DUNGEON_HEART
+; Beam effect during destruction.
+EffectBeam = EFFECTELEMENT_ELECTRIC_BALL3
+; Particle effect during destruction.
+EffectParticle = EFFECT_ELECTRIC_BALLS
+; Explosion effect after destruction.
+EffectExplosion1 = EFFECT_EXPLOSION_4
+EffectExplosion2 = EFFECT_WORD_OF_POWER
+; Used to calculate the spacing to draw the beam effect.
+EffectSpacing = 96
+; Sound played along the beam effect during destruction.
+EffectSound = 157
```

Comment:
I would do `EffectSound = 157 1` on this, like on the creature sounds or HitWaterSound of a shot. So that you could have more than a single sound on it. (the 1 being for variants)

Reply:
> I would do `EffectSound = 157 1` on this, like on the creature sounds or HitWaterSound of a shot. So that you could have more than a single sound on it. (the 1 being for variants)

ok but then how do I reflect this change on the function that play this sound?🤔

Reply:
I build it for you